BMI_GUI/app.py

```diff
@@ -0,0 +1,65 @@
+# BMI Application
+# Script to create a GUI App that calculates BMI of a person
+# Author - Sandip Dutta
+
+# Funtion to calculate the BMI, given height and weight
+# We have added height in cms and weight in Kgs (?)
+def getBMI(height : float, weight : float) -> float:
```

Comment:
Need to add facility that if user inputs measurements in other units, like ft, m etc.